### DISCUSSION TITLE
能不能加个标题翻译中文，不行我就自己加了哦

### DISCUSSION THREAD

**Discussion by 215561238** [Tyrrrz/YoutubeDownloader#266] [General]
能不能加个标题翻译中文，不行我就自己加了哦

**Comment by ScottsplaysSus** [comment]
As a guy that knows Chinese, let me reply to this :P

你好，这个程式已经进入了保养模式，应该不会添加许多新的翻译及新的作用等等，我会向程式的作者提议这个翻译的标题，谢谢你。 :)

Ok so Tyrrrz, basically the user above is asking for a Chinese translation, is it possible for us users to help and translate the app to different languages? It's definitely a good idea for multiple languages selection for better accessibilty to other users that are not fluent to English. Hope you can implement such feature for users to contribute to the translation of the app. Thanks :)

**Reply by Tyrrrz** [reply]
Thanks for the translation.
I think that would be a nice feature, but I don't know what would be a good process to update and deliver translations as new versions come out with new features and text strings. Most delivery models I've looked at were optimized for web applications, not desktop.

**Reply by ScottsplaysSus** [reply]
Alright, thanks for the reply. I hope you can find some good delivery models to assist for the translation purposes. Take your time and find out some suitable translation tool for the app :)

Chinese translation: 作者说了谢谢, 也说了现在比较难翻译单个程序, 因为大部分的翻译代码都是支持网上网站, 并不是下载在电脑的应用程序, 不过他也会尝试添加这个翻译的工具。谢谢你的建议。
